### ISSUE TITLE
[Taskbar Labels for Windows 11] Please add running indicator height customization

### ISSUE THREAD

**Issue by Anixx** [ramensoftware/windhawk-mods#952]
In mod Taskbar Labels for Windows 11 by @m417z the running indicator height cannot be customized. Even worse, it gets reset to default if set by the Taskbar Styler mod. Is there a way to customize it (for instance, set to 2 px)?

**Comment by m417z** [comment]
> it gets reset to default if set by the Taskbar Styler mod

Works for me.

![image](https://github.com/user-attachments/assets/fd6c429f-b018-4185-86a9-bb4f4abd9522)

**Comment by Anixx** [comment]
If I do this, still all the new buttons appear with thicker running indicator. This maybe because of the Taskbar Labels for Windows 11 mod interference.

**Comment by Anixx** [comment]
Or could be because I used Windows.UI.Xaml.Shapes.Rectangle#RunningIndicator. Is there any guide on what names are better?

**Comment by m417z** [comment]
> If I do this, still all the new buttons appear with thicker running indicator. This maybe because of the Taskbar Labels for Windows 11 mod interference.

I don't think so, but perhaps you're using some other software which uses XAML diagnostics, such as TranslucentTB?

**Comment by Anixx** [comment]
Also when I change VerticalAlignment on Taskbar.TaskListButton#TaskListButton it appears different on newer buttons. Do I here also use wrong object name?

**Comment by m417z** [comment]
If you'd use wrong object names, it wouldn't work for existing elements as well.

**Comment by Anixx** [comment]
But the object names are very different from yours.

**Comment by m417z** [comment]
Added an option for it in version 1.4.
